internal/pkg/manifest/validate_env.go

```diff
@@ -37,6 +37,10 @@ func (e environmentConfig) Validate() error {
 		return fmt.Errorf(`validate "http config": %w`, err)
 	}
 
+	if aws.BoolValue(e.HTTPConfig.Public.LimitToCFIngress) && !e.CDNConfig.CDNEnabled() {
+		return errors.New("CDN must be enabled to limit security group ingress to CloudFront")
```

Comment:
Can we use https://github.com/aws/copilot-cli/blob/45bdcc3549aa7a85ec3f08170ee44e5006ebe323/internal/pkg/manifest/errors.go#L57

Reply:
This doesn't quite cover the error case. The error is for when `cdn` is either not specified _or_ disabled and the `from_cdn` is enabled.

Reply:
Sorry i was about to say https://github.com/aws/copilot-cli/blob/45bdcc3549aa7a85ec3f08170ee44e5006ebe323/internal/pkg/manifest/errors.go#L75

Reply:
That doesn't work either for the same reasons 🤔 
I don't think `manifest/errors.go` has any existing error type that covers this specific validation error